app/views/caseload.html

```diff
@@ -122,22 +115,22 @@ <h3 class="heading-medium">Caseload by grade</h3>
       </tr>
     </thead>
     <tbody class="">
-      {% for team in lduCaseloadDetails %}
-        {% for i in range(0, team.grades.length) %}
+      {% for org in caseloadDetails %}
+        {% for i in range(0, org.grades.length) %}
         <tr>
           {% if i === 0 %}
-          <td rowspan="{{ team.grades.length }}" headers="name team"><a href="/team/{{ team.linkId }}">{{ team.name }}</a></td>
+          <td rowspan="{{ org.grades.length }}" headers="name team"><a href="/{{ childOrganisationLevel }}/{{ org.linkId }}">{{ org.name }}</a></td>
           {% endif %}
-          <td headers="grade team">{{ team.grades[i].gradeCode }}</td>
-          <td headers="overall tiers">{{ team.grades[i].totalCases }}%</td>
-          <td headers="untiered tiers">{{ team.grades[i].untiered }}%</td>
-          <td headers="d2 tiers">{{ team.grades[i].d2 }}%</td>
-          <td headers="d1 tiers">{{ team.grades[i].d1 }}%</td>
-          <td headers="c2 tiers">{{ team.grades[i].c2 }}%</td>
-          <td headers="c1 tiers">{{ team.grades[i].c1 }}%</td>
-          <td headers="b2 tiers">{{ team.grades[i].b2 }}%</td>
-          <td headers="b1 tiers">{{ team.grades[i].b1 }}%</td>
-          <td headers="a tiers">{{ team.grades[i].a }}%</td>
+          <td headers="grade team">{{ org.grades[i].gradeCode }}</td>
+          <td headers="overall tiers">{{ ((org.grades[i].totalCases * 10)  | round) / 10 }}%</td>
```

Comment:
Is there a way to write this rounding as a callable function?

Reply:
changed